target/linux/x86/geode/config-6.6

```diff
@@ -35,43 +35,36 @@ CONFIG_ACPI_THERMAL=y
 # CONFIG_ACPI_WMI is not set
 # CONFIG_ADV_SWBUTTON is not set
 CONFIG_ALIX=y
-# CONFIG_AMD_PMC is not set
 # CONFIG_AMD_PMF is not set
-CONFIG_ARCH_MIGHT_HAVE_ACPI_PDC=y
 # CONFIG_ASUS_TF103C_DOCK is not set
+CONFIG_ARCH_MIGHT_HAVE_ACPI_PDC=y
```

Comment:
I agree using ASCII ordering is better.  Alas this one the order became wrong.

Reply:
fixed